drivers/framework.go

```diff
@@ -12,7 +12,7 @@ var Drivers map[string]WebhookDriver
 //WebhookDriver interface for all drivers
 type WebhookDriver interface {
 	ValidatePayload(config interface{}, apiClient *client.RancherClient) (int, error)
-	Execute(config interface{}, apiClient *client.RancherClient, requestBody interface{}) (int, error)
+	Execute(config interface{}, apiClient *client.RancherClient, request interface{}) (int, error)
```

Comment:
Separate the change of `requestBody` to `request` into a separate commit.

It's a separate logic from adding a new webhook

Reply:
I can't recall why we made the field a generic map before but now, since we are always passing an http.Request, let's just make that the type on the interface.

Reply:
@biblesyme ^^

Reply:
Gotcha